MediaBrowser.Providers/Manager/ItemImageProvider.cs

```diff
@@ -114,11 +139,22 @@ public bool ValidateImages(BaseItem item, IEnumerable<IImageProvider> providers,
                 }
             }
 
+            // only delete existing multi-images if new ones were added
+            if (oldBackdropImages.Count > 0 && oldBackdropImages.Count < item.GetImages(ImageType.Backdrop).Count())
+            {
+                PruneImages(item, oldBackdropImages);
```

Comment:
What if some of the new images have overwritten the old ones? Can that happen?

Reply:
It cannot: the image saver always adds at a new index when `item.AllowsMultipleImages` and the checks to update in place only trigger when the same index is used.

While reviewing the code with your question in mind I did find a very specific edge case where backgrounds in both the item and the remote provider might not be re-downloaded and would then be pruned after updates completed, but I couldn't trigger it testing against live data. I pushed the fix for that and I'll add a test case tomorrow to exercise that path and prove it works as intended.